akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowLogSpec.scala

```diff
@@ -28,7 +28,7 @@ class FlowLogSpec extends AkkaSpec("akka.loglevel = DEBUG") with ScriptedTest {
 
   "A Log" must {
 
-    val supervisorPath = "akka://FlowLogSpec/user/$a"
+    val supervisorPath = "akka://FlowLogSpec/user/StreamSupervisor-0"
```

Comment:
How would that work? The ActorMaterializer wraps an ActorRefFactory, i.e. it will use `actorOf` on that context, or in other words, it will create this supervisor actor as the child of the actor that created the materializer or the top level system.